src/renderer/components/Preference/Basic.vue

```diff
@@ -67,11 +67,11 @@
           </el-col>
         </el-form-item>
         <el-form-item
```

Comment:
L69-77
The position of the auto hide window setting in the preferences should be moved to the appearance group.